src/test/java/org/cactoos/http/HtHeadTest.java

```diff
@@ -106,4 +108,52 @@ public void largeText() throws IOException {
         );
     }
 
+    @Test
+    public void edgeOfTheBlockTearing() throws IOException {
+        final int size = 16384;
+        final String header = new JoinedText(
+            "\r\n",
+            "HTTP/1.1 200 OK",
+            "Referer: http://en.wikipedia.org/wiki/Main_Page#\0",
+            "Content-type: text/plain",
+            ""
+        ).asString();
+        final String block = header.replace(
```

Comment:
@ilyakharlamov use `ReplacedText` from cactoos instead if possible and avoid calling `asStsring()` when possible

Reply:
@victornoel  agreed, done in the commit adc5719